server/client_manager.py

```diff
@@ -106,52 +113,57 @@ def change_area(self, area):
+            self.send_command('LE', *self.area.get_evidence_list(self))
 
         def send_area_list(self):
             msg = '=== Areas ==='
+            lock = {True: '[LOCKED]', False: ''}
             for i, area in enumerate(self.server.area_manager.areas):
-                msg += '\r\nArea {}: {} (users: {})'.format(i, area.name, len(area.clients))
-                if self.area == area:
-                    msg += ' [*]'
-                msg += '\r\n[{}]'.format(area.status)
-                if area.is_locked:
-                    msg += '[LOCKED]'
-            self.send_host_message(msg)
-
-        def send_limited_area_list(self):
-            msg = '=== Areas ==='
-            for i, area in enumerate(self.server.area_manager.areas):
-                msg += '\r\nArea {}: {}'.format(i, area.name)
+                owner = 'FREE'
+                if area.owned:
+                    for client in area.clients:
+                        if client.is_cm:
+                            owner = 'MASTER: {}'.format(client.get_char_name())
+                            break
+                msg += '\r\nArea {}: {} (users: {}) [{}][{}]{}'.format(i, area.name, len(area.clients), area.status, owner, lock[area.is_locked])
                 if self.area == area:
                     msg += ' [*]'
             self.send_host_message(msg)
 
-        def get_area_info(self, area_id):
+        def get_area_info(self, area_id, mods):
             info = ''
             try:
                 area = self.server.area_manager.get_area_by_id(area_id)
             except AreaError:
                 raise
             info += '= Area {}: {} =='.format(area.id, area.name)
-            sorted_clients = sorted(area.clients, key=lambda x: x.get_char_name())
+            sorted_clients = []
+            for client in area.clients:
+                if (not mods) or client.is_mod:
+                    sorted_clients.append(client)
+            sorted_clients = sorted(sorted_clients, key=lambda x: x.get_char_name())
             for c in sorted_clients:
-                info += '\r\n{}'.format(c.get_char_name())
+                info += '\r\n[{}] {}'.format(c.id, c.get_char_name())
+                if self.is_mod:
+                    info += ' ({})'.format(c.ipid)
             return info
 
-        def send_area_info(self, area_id):
-            try:
-                info = self.get_area_info(area_id)
-            except AreaError:
-                raise
-            self.send_host_message(info)
-
-        def send_all_area_info(self):
-            info = '== Area List =='
-            for i in range(len(self.server.area_manager.areas)):
-                #print(len(i.clients))
-                if len(self.server.area_manager.areas[i].clients) > 0:
-                    info += '\r\n{}'.format(self.get_area_info(i))
+        def send_area_info(self, area_id, mods): #if area_id is -1 then return all areas. If mods is True then return only mods
```

Comment:
Use docstrings instead of putting comments on the same line as the method declaration.